Stellarium/stellarium
# Fix: unambiguous comet names

### Description




Some recent changes have broken the solar system loading a bit, in that the object name is too short, and data for recurring comets like Halley are overwritten. 

Branch fix/3349 provides a different solution which adds another instance variable to the Comet class. I don't think it's required.

This alternative PR continues re-writing the data files on the one hand, and intends to create unambiguous names on loading.
Another test during load should ensure that data are not overwritten.

To complete that work:
- [x] Adapt SSE plugin accordingly.
- [x] Re-format 1000_comets.ini.

Fixes #3349 (issue)

### Screenshots (if appropriate):

### Type of change


### How Has This Been Tested?




Usually, for testing changes: 
- remove current ssysstem_minor.ini from user data dir
- launch program
- import 1000_comets.ini
- Search for Halley and see all entries listed.

**Test Configuration**:
* Operating system: Windows 11
* Graphics Card: irrelevant

## Checklist:
